app/sections/view-implementation-timescales/macro.test.js

```diff
@@ -0,0 +1,73 @@
+import request from 'supertest';
+import cheerio from 'cheerio';
+import { createTestHarness } from '../../testUtils/testHarness';
+
+const macroWrapper = `{% from './sections/view-implementation-timescales/macro.njk' import viewImplementationTimescales %}
+                        {{ viewImplementationTimescales(params) }}`;
+
+describe('view-implementation-timescales', () => {
+  it('should render the title of the section', (done) => {
+    const context = {
```

Comment:
you could put the context in a const at the top of the tests to avoid duplication

Reply:
or even better - pull it in from the settings file